code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesActivity.java

```diff
@@ -32,18 +33,20 @@
 public class PlacesActivity extends Activity {
 
 	private static final String FINE_LOCATION = Manifest.permission.ACCESS_FINE_LOCATION;
+	private static final String BACKGROUND_LOCATION = Manifest.permission.ACCESS_BACKGROUND_LOCATION;
+	private static final String INTENT_PERMISSION_KEY = "intent permission key";
 
 	/**
-	 * Checks if the permission to access fine location is granted.
+	 * Checks if the permission to access fine location during app in use is granted.
```

Comment:
`Checks if permission to access fine location while app is in use has been granted.`